examples/CMakeLists.txt

```
@@ -1,25 +1,35 @@
 option(GINKGO_BUILD_EXTLIB_EXAMPLE "Build the external-lib-interfacing with deal.II, you need to link the deal.II library." OFF)
-add_subdirectory(custom-logger)
-add_subdirectory(custom-matrix-format)
-add_subdirectory(custom-stopping-criterion)
+option(GINKGO_RUN_EXAMPLES "Run the examples." OFF)
+
+set(EXAMPLES_EXEC_LIST "adaptiveprecision-blockjacobi;custom-logger;ginkgo-overhead;ginkgo-ranges;ilu-preconditioned-solver;ir-ilu-preconditioned-solver;inverse-iteration;iterative-refinement;mixed-precision-ir;nine-pt-stencil-solver;poisson-solver;preconditioned-solver;simple-solver;three-pt-stencil-solver;")
```

I would suggest listing one example per line, this makes for nicer diffs in the future.
```suggestion
set(EXAMPLES_EXEC_LIST
    adaptiveprecision-blockjacobi
    custom-logger
    ginkgo-overhead
    ginkgo-ranges
    ilu-preconditioned-solver
    ir-ilu-preconditioned-solver
    inverse-iteration
    iterative-refinement
    mixed-precision-ir
    nine-pt-stencil-solver
    poisson-solver
    preconditioned-solver
    simple-solver
    three-pt-stencil-solver)
```

---

Is this a list of examples which can take in executor as parameter? If so, I believe at least `simple-solver-logging` can be here as well.

---

Yes, but `simple-solver-logging` prints out pointers and they can be different on each run.